apps/dev/package.json

```diff
@@ -19,7 +19,7 @@
     "@next-auth/typeorm-legacy-adapter": "workspace:*",
     "@prisma/client": "^3",
     "faunadb": "^4",
-    "next": "12.3.1",
+    "next": "13.0.1",
```

Comment:
Do you know why an exact version is being used here? Why not use "^13" as it's done for the other dependencies?

Reply:
I just wanted to make sure to get Next.js Version `13.0.1`, because there was a breaking change between Next.js `13.0.0` and `13.0.1`. This means we can't rely on semver conventions here.

Nevertheless since this is a dev app you can change this when developing to fit your needs.

Reply:
Brilliant, thanks